go.mod

```diff
@@ -27,6 +27,7 @@ require (
 	gomodules.xyz/jsonpatch/v2 v2.1.0
 	google.golang.org/api v0.25.0
 	gopkg.in/yaml.v2 v2.3.0 // indirect
+	gotest.tools v2.2.0+incompatible
```

Comment:
This doesn't seems right :thinking:

Reply:
That's right. I'm replacing it to use what's already being used in this repo

Reply:
Removed 😄